Firestore/core/src/remote/bloom_filter.h

```diff
@@ -63,6 +82,14 @@ class BloomFilter final {
 
   // Bloom filter's bitmap.
   std::vector<uint8_t> bitmap_;
+
+  BloomFilter::Hash Md5HashDigest(const absl::string_view key) const;
```

Comment:
nit: Remove `const` from `const absl::string_view key`. Since the string_view is being passed in by value there is no need to declare it `const`.

Reply:
make sense, lemme pass it by reference instead.

Reply:
absl:string_view should _always_ be passed by value. It is a special class that avoids making inexpensive copies. It's an exception to the rule that passing by value for "complex" objects is "cheaper".